src/index.js

```diff
@@ -738,9 +742,17 @@ if (!('attachShadow' in document.createElement('div'))) {
     // Polyfill as much as we can and work around WebKit in other areas.
     if (canPatchNativeAccessors || polyfillAtRuntime.indexOf(memberName) === -1) {
       const nativeDescriptor = findDescriptorFor(memberName);
+      const isDefinedInTextProto = memberName in textProto;
+      const shouldOverrideInTextNode = doNotOverridePropertiesInTextNodes.indexOf(memberName) === -1;
```

Comment:
If this was moved down to underneath Object.defineProperty, it'd make all the logic in one place, which might be more readable.